components/Banner/examples/supplemental/MDCBannerView.h

```diff
@@ -18,15 +18,42 @@
 
```

Comment:
Please import MaterialButton.h since it appears you rely on the class. I believe we expect cross-component dependencies to be imported where they are required.

Reply:
Addressed in 7ab34a3d75a627821b8dde16c3ca02660a1d615b